ui/app/controllers/vault/cluster/auth.js

```diff
@@ -7,16 +8,36 @@ export default Controller.extend({
   vaultController: controller('vault'),
   clusterController: controller('vault.cluster'),
   namespaceService: service('namespace'),
+  featureFlagService: service('featureFlag'),
   namespaceQueryParam: alias('clusterController.namespaceQueryParam'),
   queryParams: [{ authMethod: 'with' }],
   wrappedToken: alias('vaultController.wrappedToken'),
   authMethod: '',
   redirectTo: alias('vaultController.redirectTo'),
+  managedNamespaceRoot: alias('featureFlagService.managedNamespaceRoot'),
+
+  managedNamespaceChild: computed('namespaceQueryParam', function() {
```

Comment:
Had to change these from getters to computed properties so they would work properly in this version of Ember